colors/base16-solarized-dark.vim

```diff
@@ -249,6 +249,7 @@ call <sid>hi("Constant",     s:gui09, "", s:cterm09, "", "", "")
 call <sid>hi("Define",       s:gui0E, "", s:cterm0E, "", "none", "")
 call <sid>hi("Delimiter",    s:gui0F, "", s:cterm0F, "", "", "")
 call <sid>hi("Float",        s:gui09, "", s:cterm09, "", "", "")
+call <sid>hi("NormalFloat",  s:gui0A, s:gui01, s:cterm0A, s:cterm01, "", "")
```

Comment:
The only thing in this line I'm sure of is the `s:cterm01` part

Reply:
Fixing it in https://github.com/fnune/base16-vim/pull/40